apps/dolly-backend/src/main/java/no/nav/dolly/repository/OrganisasjonBestillingMalRepository.java

```diff
@@ -0,0 +1,28 @@
+package no.nav.dolly.repository;
+
+import no.nav.dolly.domain.jpa.Bruker;
+import no.nav.dolly.domain.jpa.OrganisasjonBestillingMal;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+public interface OrganisasjonBestillingMalRepository extends CrudRepository<OrganisasjonBestillingMal, Long> {
+
+    List<OrganisasjonBestillingMal> findByIdContaining(String id);
+
+    List<OrganisasjonBestillingMal> findByBrukerAndMalBestillingNavn(Bruker bruker, String navn);
+
+    List<OrganisasjonBestillingMal> findByBruker(Bruker bruker);
+
+    @Query(value = "from OrganisasjonBestillingMal b where b.malBestillingNavn is not null order by b.malBestillingNavn")
+    List<OrganisasjonBestillingMal> findMalBestilling();
+
+    @Transactional
+    @Modifying
+    @Query("update BestillingMal b set b.malBestillingNavn = ?2 where b.id = ?1")
```

Comment:
Hva med å benytte navngitte parametere i stedet for posisjonsbestemte? Det øker lesbarhet. 😃